cmd/storage.go

```diff
@@ -52,16 +52,22 @@ var storageCreateCmd = &cobra.Command{
 }
 
 var storageUnmountCmd = &cobra.Command{
-	Use:   "unmount [storage name]",
-	Short: "Unmount storage from the current component",
-	Long: `Unmount storage from the current component.
+	Use:   "unmount [storage/path name]",
+	Short: "Unmount storage or storage with the given path, from the current component",
+	Long: `Unmount storage or storage with the given path, from the current component.
```

Comment:
How about this:
Unmount storage from the given path or unmount storage identified by its name.